configure.ac

```diff
@@ -1,5 +1,5 @@
 AC_PREREQ(2.59)
-AC_INIT([singularity],[2.4.5],[gmkurtzer@gmail.com])
+AC_INIT([singularity],[2.4.6],[gmkurtzer@gmail.com])
```

Comment:
I think this really needs to be 2.4.6rc2 because of all of the potential problems that others have noted.  I know that this may cause some challenges with release numbers, but at least folks will actually see what version they are running using the `--version` command.

Reply:
Version for me is:
```
$ singularity --version
2.4.6-ver_fix.gcef599d
```

So the branch (ver_fix) is included as well as the git commit. If we have the 'rc' included, then a RPM update wont work because `2.4.6 < 2.4.6rc`.

For _testing_ it should be being built from the git tree and not a dist tarball anyway. It's not meant to stick around.

Reply:
> So the branch (ver_fix) is included as well as the git commit. 
Well, sure.  But a soon as we make the 2.4.6-rc2 tag, this thing well return...
```
$ singularity --version
2.5.6-dist
```
That's a problem because now a user thinks they have the real deal when they actually mistakenly installed a release candidate and we can't tell the difference if they send us a bug report either.  

As far as the update issue, if somebody has created and installed an RPM from a release candidate then presumably they are savvy enough to know how to remove the release candidate and install the release.  And if they forget or don't know how the `--version` command should be able to tell them that they don't have the latest release.  Right?

Reply:
>  And if they forget or don't know how the --version command should be able to tell them that they don't have the latest release.

No. It will show:  `2.4.6-dist` or `2.5.0-dist` when building from a tarball (i.e. creating a package). When you do `make dist` the `.git/` directory isn't there, so it doesn't look up the git information and version becomes:  `@PACKAGE_VERSION@-dist` instead of `@PACKAGE_VERSION@-[branch](-[git commit])`

Look at `configure.ac` ~line 314